### PR TITLE
Fixes API for lder version without BuildTools, adds tests for API v5

### PR DESCRIPTION
**Pull request by Karm**

### CODE REVIEW

File: src/main/java/org/eclipse/microprofile/starter/rest/APIEndpointV6.java [hunk 1]
```diff
@@ -19,30 +19,12 @@
  */
 package org.eclipse.microprofile.starter.rest;
 
-import javax.inject.Inject;
```
Comment (Emily-Jiang) [inline src/main/java/org/eclipse/microprofile/starter/rest/APIEndpointV6.java:22]: Why did you delete the block? It is literally a blank java file.

Comment (Emily-Jiang) [reply src/main/java/org/eclipse/microprofile/starter/rest/APIEndpointV6.java:22]: I see you make this file as the latest.

**Comment by Emily-Jiang** [comment]
@Karm you PR looks good to me. I see you put all latest info to the default json and update the previous version to only support `MAVEN`. I saw Kumuluzee had test failure. I re-run all tests to see whether the failure was just a glitch.

File: src/main/java/org/eclipse/microprofile/starter/rest/APIEndpointV6.java (hunk 1 above)
Comment (Karm) [reply src/main/java/org/eclipse/microprofile/starter/rest/APIEndpointV6.java:22]: @Emily-Jiang The bug was a result of a misunderstanding. The way it is setup now ( and I am not saying that it is the best way to do it, but it is what we have) is that each APIEndpointV* lists *differences* in methods compared to the latest.
Hence Version 6 is empty now as it has no differences.
When someone goes and touches the current API in latest, he/she should also edit the V6 file and add the difference there.
Then a new V7 file will be empty and  /api/ will point to to the same as /api/7/....

**Comment by Karm** [comment]
> @Karm you PR looks good to me. I see you put all latest info to the default json and update the previous version to only support `MAVEN`. I saw Kumuluzee had test failure. I re-run all tests to see whether the failure was just a glitch.

Liberty sometimes glitches, but Kumuluzee fails always on: 

```
[INFO] Unpacking kumuluzee-loader dependency.
[INFO] Building jar: /home/runner/work/_temp/kumuluzee/target/kumuluzee-uber.jar
[INFO] Repackaging jar: /home/runner/work/_temp/kumuluzee/target/kumuluzee.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  4.774 s
[INFO] Finished at: 2021-09-08T21:24:45Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.kumuluz.ee:kumuluzee-maven-plugin:3.9.0:repackage (package) on project kumuluzee: Execution package of goal com.kumuluz.ee:kumuluzee-maven-plugin:3.9.0:repackage failed.: UnsupportedOperationException -> [Help 1]

```

...and I don't know why and I cannot reproduce it locally. I tried also locally empty maven repo to let it download everything. 
I also tried OpenJDK 11.0.13_5 EA.  No luck. It runs just fine locally.

**Comment by Karm** [comment]
@Emily-Jiang I tried to enforce Maven 3.6.3....let's see...

**Comment by Karm** [comment]
Looks good. Kumuluzee is fixed.   

@Emily-Jiang   Ack  / NAck on this PR?
